packages/grafana-ui/src/components/VizTooltip/SeriesList.tsx

```diff
@@ -18,13 +14,18 @@ export const SeriesList = ({ series }: SeriesListProps) => {
   return (
     <>
       {series.map((series, index) => {
+        // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+        const label = series.label as string;
```

Comment:
Hi all! I have types clashing. The question is about the Trend tooltip. Do we really need [SeriesList](https://github.com/grafana/grafana/pull/77251/files#diff-ce7350857bc47be428da070d7d0c7549e20d4c3b06ca260fedc574c8e04bd520R147) component? It looks totally replaceable by `VizTooltipRow` component I implemented here.
I noticed that in SeriesList the label can be `label?: React.ReactNode` type. Are there any scenarios we have to render ReactNode in a label instead of justa string?
Can we take a look at this part together?

cc @adela-almasan

Reply:
Hey! [Context PR](https://github.com/grafana/grafana/pull/44774) for the `React.ReactNode` type - probably we'll get the chance to test more with complex cases with timeseries. I think `SeriesList` looks like a wrapper for `VizTooltipRow` now, so that should be fine - we can iterate depending on how things evolve.